src/common/src/types/mod.rs

```diff
@@ -242,6 +242,25 @@ impl DataType {
             List { datatype } => datatype.mem_cmp_eq_value_enc(),
         }
     }
+
+    /// Check if the datatype has fixed length.
+    pub fn has_fixed_length(&self) -> bool {
```

Comment:
Where is this used?

Reply:
I used this for deserialization, but we no longer need it now😇

Reply:
removed